README.md

```diff
@@ -61,31 +60,38 @@ These are the available configuration options:
           "VolumeDown": true
         }
       },
-      "macros": {
-        "Play Xbox 360": [
-          [ "gpio", "TV", 1],
-          [ "gpio", "Receiver", 1],
-          [ "gpio", "Xbox", 1],
-          [ "SonyTV", "Power" ],
-          [ "delay", 500 ],
-          [ "SonyTV", "Xbox360" ],
-          [ "Yamaha", "Power" ],
-          [ "delay", 250 ],
-          [ "Yamaha", "Xbox360" ],
-          [ "Xbox360", "Power" ]
-        ],
-        "Listen to Music": [
-          [ "gpio", "Receiver", 1],
-          [ "Yamaha", "Power" ],
-          [ "delay", 500 ],
-          [ "Yamaha", "AirPlay" ]
-        ],
-        "all off": [
-          [ "gpio", "TV", 0],
-          [ "gpio", "Receiver", 0],
-          [ "gpio", "Xbox", 0]
-        ],
-      },
+      "macros": [
+        {
+          "name": "Xbox360",
+          "sequence": [
+            [ "gpio", "TV", 1 ],
+            [ "gpio", "Receiver", 1 ],
+            [ "gpio", "Xbox", 1 ],
+            [ "SonyTV", "Power" ],
+            [ "SonyTV", "Xbox360" ],
+            [ "Yamaha", "Power" ],
+            [ "Yamaha", "Xbox360" ],
+            [ "Xbox360", "Power" ]
+          ]},
+        {
+          "name": "lights off",
+          "defaultDelay": 40,
+          "sequence": [
+            [ "Lightcontrol", "C01off"],
+            [ "Lightcontrol", "C02off"],
+            [ "Lightcontrol", "C03off"]
+          ]},
+        {
+          "name": "all off",
+          "defaultDelay": 20,
+          "sequence": [
+            [ "call", "lights off"],
```

Comment:
I believe that if this was called `macro` it would be more intuitive.

Reply:
since the sequence is just a property of the macro, but not the macro itself, it would lead to much less intuitive expressions on the code side. I'd keep it.